### PR TITLE
Sample task data for the PS algorithm

### PR DESCRIPTION
**Pull request by gstuder-ona**
... along with images for the README, etc.

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (gstuder-ona) [inline .pre-commit-config.yaml:29]: black sometimes puts binary ops (aka "+") on their own line, so flake8 probably just needs to deal with it

File: README.md
Comment (gstuder-ona) [inline README.md]: I don't have a good process for getting these links working - do I need to edit them in the PR, or do I need to wait until stuff is merged to master so the image link is stable?

File: chti/abnormal_task_perf_analyzer.py
Comment (gstuder-ona) [inline chti/abnormal_task_perf_analyzer.py:552]: This is definitely something to look at @pld @lincmba, as part of sanity-checking the sample data I noticed that we weren't scaling the abnormality contributions of each feature (aka task) by the actual difference-from-mean for each worker.  That means that we were just figuring out which features *could* have made things abnormal but not the ones that actually were for the particular worker.  Intuitively this means that one abnormal task would "drag" otherwise normal tasks (for that worker) to look abnormal if there were other workers who had those tasks as abnormal too.

File: README.md
Comment (pld) [reply README.md]: what I do is push something that is broken until after merge, for this

https://raw.githubusercontent.com/onaio/community-health-task-investigator/master/docs/images/task_time_dist_1.png

File: chti/abnormal_task_perf_analyzer.py
Comment (gstuder-ona) [reply chti/abnormal_task_perf_analyzer.py:552]: There's also a second, unrelated issue I also found (need to open a ticket) - because our task features are "surprise" and we're just looking for any abnormal surprise, the analysis is also picking up cases where surprise is abnormally *low* (aka the worker matches the historical model exactly).  I'm not sure we want this considered in the results.  In the real data sets we tried this was dwarfed by any kind of surprising results, but it's easy to see in the synthetic data set when the analyzed worker task data is drawn from the exact same distribution as the historical worker task data.

**Review by pld — APPROVED** [review]

**Pull request merged**
